instadam/project.py

```diff
@@ -319,3 +324,39 @@ def update_user_permission(project_id):
         db.session.commit()
 
     return construct_msg('Permission added successfully'), 200
+
+
+@bp.route('/project/<project_id>', methods=['DELETE'])
+@jwt_required
+def delete_user(project_id):
+    project = maybe_get_project(project_id)  # check privilege and get project
+
+    # Delete images
```

Comment:
Your other comments are good, but I don't think you need these delete comments because the code is short and clear

Reply:
Acknowledged